apps/bezier-react/src/components/KeyValueListItem/common/KeyItem/KeyItem.styled.ts

```diff
@@ -0,0 +1,23 @@
+/* Internal dependencies */
+import { ellipsis, styled } from 'Foundation'
+import { Text } from 'Components/Text'
+import { Stack, StackItem } from 'Components/Stack'
+
+export const KeyContentStack = styled(Stack).attrs({
+  direction: 'horizontal',
+  align: 'center',
+  spacing: 8,
+})`
+  min-width: 100px;
+`
+
+export const KeyText = styled(Text).attrs(() => ({
+  forwardedAs: 'div',
+}))`
+  color: ${({ foundation }) => foundation?.theme?.['txt-black-dark']};
+  ${ellipsis(1)}
+`
+
+export const KeyIconStackItem = styled(StackItem)``
+
+export const KeyTextStackItem = styled(StackItem)``
```

Comment:
```suggestion
export const KeyIconStackItem = styled(StackItem).attrs({
  size: 20,
})``

export const KeyTextStackItem = styled(StackItem).attrs({
  grow: true,
  shrink: true,
  weight: 1,
})`
  ${ellipsis(1)}
`
```

**AS-IS**
<img width="349" alt="스크린샷 2022-05-24 오후 3 35 26" src="https://user-images.githubusercontent.com/25701854/169964447-95bc9f5b-539f-44cb-996d-282294e4f750.png">

**TO-BE**
<img width="340" alt="스크린샷 2022-05-24 오후 3 35 07" src="https://user-images.githubusercontent.com/25701854/169964411-9e418a27-d851-4928-a7ea-7668f84f3bdd.png">